apps/vr-tests/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -1,49 +1,49 @@
 import * as React from 'react';
-import { DecoratorFunction } from '@storybook/addons';
-import { ExtendedStoryFnReturnType } from './types';
+import type { DecoratorFn } from '@storybook/react';
 
-export const TestWrapperDecorator: DecoratorFunction<ExtendedStoryFnReturnType> = story => (
+export const TestWrapperDecorator: DecoratorFn = (story, context) => (
   <div style={{ display: 'flex' }}>
     <div className="testWrapper" style={{ padding: '10px', overflow: 'hidden' }}>
-      {story()}
+      {story(context)}
     </div>
   </div>
 );
 
-export const TestWrapperDecoratorTall: DecoratorFunction<ExtendedStoryFnReturnType> = story => (
+export const TestWrapperDecoratorTall: DecoratorFn = (story, context) => (
   <div style={{ display: 'flex' }}>
     <div className="testWrapper" style={{ padding: '10px 10px 120px' }}>
-      {story()}
+      {story(context)}
     </div>
   </div>
 );
 
-export const TestWrapperDecoratorTallFixedWidth: DecoratorFunction<
-  ExtendedStoryFnReturnType
-> = story => (
+export const TestWrapperDecoratorTallFixedWidth: DecoratorFn = (story, context) => (
   <div style={{ display: 'flex' }}>
     <div className="testWrapper" style={{ padding: '10px 10px 120px', width: '300px' }}>
-      {story()}
+      {story(context)}
     </div>
   </div>
 );
 
-export const TestWrapperDecoratorFixedWidth: DecoratorFunction<
-  ExtendedStoryFnReturnType
-> = story => (
+export const TestWrapperDecoratorFixedWidth: DecoratorFn = (story, context) => (
   <div style={{ display: 'flex' }}>
-    <div className="testWrapper" style={{ padding: '10px', width: '300px' }}>
-      {story()}
+    <div id="testWrapper" className="testWrapper" style={{ padding: '10px', width: '300px' }}>
```

Comment:
last thing/question: why do we need to add that `id` attribute ? if there is no reason please remove. ty

Reply:
It was needed for some story, but I already removed it in one of previous commits. The diff is outdated.